app/views/form/guidance/_what_counts_as_income.html.erb

```diff
@@ -0,0 +1,25 @@
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      What counts as income?
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+
+  <p class="govuk-body">You should include any income from:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>employment</li>
+    <li>pensions</li>
+    <li>Universal Credit</li>
+  </ul>
+
+  <p class="govuk-body">Don’t include:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>National Insurance (NI) contributions and tax</li>
+    <li>housing benefit</li>
+    <li>child benefit</li>
+    <li>council tax support</li>
+  </ul>
+
+  </div>
+</details>
```

Comment:
Does this have to be HTML as it gets rendered as JSON? Be nice if could use a rails component(s) here, if possible.

Reply:
What do you mean it gets rendered as JSON? How so?

Reply:
I might have been overthinking my comment! We can do the following, no?

```erb
<%= govuk_details(summary_text: 'What counts as income?') do %>
  <p class="govuk-body">You should include any income from:</p>
  <ul class="govuk-list govuk-list--bullet">
    <li>employment</li>
    <li>pensions</li>
    <li>Universal Credit</li>
  </ul>

  <p class="govuk-body">Don’t include:</p>
  <ul class="govuk-list govuk-list--bullet">
    <li>National Insurance (NI) contributions and tax</li>
    <li>housing benefit</li>
    <li>child benefit</li>
    <li>council tax support</li>
  </ul>
<% end %>
```

Reply:
Ah yes sorry, the rendering thing threw me but we should definitely use the component. Have updated.